packages/@aws-cdk/aws-secretsmanager/lib/rds-rotation-single-user.ts

```diff
@@ -0,0 +1,204 @@
+import ec2 = require('@aws-cdk/aws-ec2');
```

Comment:
Feels like this whole file should live in the `aws-rds` package.

Reply:
OK, to move the RDS rotation, in the same PR then? It breaks the conventional commit + squash convention here...

Reply:
Not sure what you mean. The only thing it might break is the scope, if the scope were a single package.

I would feel comfortable with a `feat(secretsmanager/rds): support credential rotation`.